src/App.js

```diff
@@ -42,7 +42,7 @@ class App extends Component {
     for (let i = 0; i < this.state.technologies.length; i++) {
       rows.push(
         <p key={this.state.technologies[i].name} style={this.rowStyle(this.state.technologies[i].name)}>
-          <strong>{this.state.technologies[i].name}</strong> has been out for <strong>{this.daysSince(this.state.technologies[i].released)} years</strong>
+          <strong>{this.state.technologies[i].name}</strong> has been out for <strong>{this.daysSince(this.state.technologies[i].released)} year{(this.daysSince(this.state.technologies[i].released) !== 1) ? "s":""}</strong>
         </p>
```

Comment:
I think it would be applicable to 0 too.

Reply:
I thought about/tried that and I don't think so. 

"This project has been out for 0 year" vs  "This project has been out for 0 years" 
The first one is not grammatically correct.

Reply:
An alternative could be "This project has been out for <1 year", but if we're sticking with "0", it should be "years".

Reply:
I like "<1 year" approach more

Reply:
I agree with @umer936 this looks better than including an additional dependency, @disissid there is also #75 so, with that being said let me know which one to merge and also I believe too that this should be "<1 year" or just plain "less than 1 year"

Reply:
I like the format "less than a year" since it's quite conversational and suits the overall style of the page.

Reply:
@Esteban-Rocha I've updated #75 for 0 year case